### ISSUE TITLE
Home, 404 error at the end of the installation

### ISSUE THREAD

**Issue by Novaslash** [PrestaShop/PrestaShop#22037]
Hello,

First of all, please excuse me for my English.

I am using Prestashop version 1.7.6.8.

After working locally on my personal theme, I decided to send everything to the server first without my theme.

The installation went without a hitch, but while the administration seems to be running smoothly, the store does not. Indeed, from the reception, the link remains blocked on "https://dnoi.fr/store/install/"

However, I have deleted the "install" folder as indicated. If I test a category, I get the message "The page is not redirected correctly".

The problem arose from the start so without any modification on the administration side.

I specify that I did not modify any of the base files of Prestashop. I only copied the Classic theme files that I needed into my personal theme.

My htaccess file is in chmod 777. I have also deactivated and then reactivated the simplified urls to recreate the htacess without success.

If I try to resend the install folder, Prestashop starts a new installation for me. I tried but nothing changed.

In Traffic & SEO> URL configuration, the configuration seems correct:

domain: dnoi.fr
domain_ssl: dnoi.fr
physical_uri: / store /

The store is actually in the "store" folder

In Redirect to Canonical URL, I tried in 301 (instead of 302 by default) but it didn't change anything.

Emptying the cache has no effect.

However, two or three weeks ago, I had tested a 1st online installation before any modifications (to be sure not to work for nothing) to create my theme and everything worked (I just had to downgrade the version of php in 7.1 at my host), I didn't have this 404 error problem at all.

The files sent are only the originals so without any modification.

I'm completely lost. Already that I am not an expert but there, I understand absolutely nothing knowing that the very first time does not pose any problems to me.

Can you help me ?

Thank you

Edit : I forgot that locally, with Wamp, I have no problems. This information may not be very useful to you, but just in case

Edit 2 : Edit: I finally have an improvement. In the BO, I activated "Activate SSL" as well as "Activate SSL on the whole site". It seems that all links (category, footer, ...) are working now. The last worry I have left is the reception. Whether I enter the url myself or click on the logo, I'm always redirected to this 404 error about the install folder.

**Comment by SimonGrn** [comment]
Hello,
Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template to submit a complete bug report**. 
Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it. 

Thank you
